src/elements/content-sidebar/activity-feed/task-new/TaskDueDate.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedTime, FormattedMessage } from 'react-intl';
+
+import messages from '../../../common/messages';
+import { TASK_NEW_INCOMPLETE } from '../../../../constants';
+
+type Props = {
+    dueDate?: ?ISODate,
+    status: TaskStatus,
+};
+
+const TaskDueDate = ({ dueDate, status }: Props): React.Node => {
```

Comment:
It looks like this logic wasn't being tested before, since no tests were moved over. If you have time to write some, we'd appreciate it.

Reply:
It's partially covered by the generic `Task` snapshot since that was where it was before, but the internal content/structure isn't tested. I can one or two for that

Reply:
Isn't it tested here? https://github.com/box/box-ui-elements/pull/1046/files#diff-0107f674890feab34adb2be02935445fR87